### ISSUE TITLE
Crashes under Linux

### ISSUE THREAD

**Issue by ghost** [Tricky1975/BallPlayCupid#75]
It crashed for me under Linux as I pressed a key (the left arrow, I think) when I entered the game the first time.

```
Each received a nil!    255 0   0
Error: SCRIPT/GAME.LLL/GAME.LUA:848: attempt to call a nil value
stack traceback:
    SCRIPT/GAME.LLL/GAME.LUA:848: in function 'keypressed'
    LIBS/CHAIN.LLL/CHAIN.LUA:60: in function <LIBS/CHAIN.LLL/CHAIN.LUA:55>
    [string "boot.lua"]:454: in function <[string "boot.lua"]:435>
    [C]: in function 'xpcall'
```

And then it crashes without even the first loading screen, every time I try to enter again, unless I delete the save directory:

```
$ love10 BallPlayCupid
Love save=<redacted>/.local/share/love/BallPlayCupid
Error: main.lua:79: Import runtime error:SCRIPT/ILOVEYOU.LUA
main.lua:79: Import runtime error:SCRIPT/TITLE.LUA
LIBS/SAVE.LLL/SAVE.LUA:78: save.load('GLOB/LOGIN.LUA',nil,nil): attempt to call a nil value
stack traceback:
    [C]: in function 'assert'
    main.lua:79: in function 'j_love_import'
    main.lua:346: in main chunk
    [C]: in function 'require'
    [string "boot.lua"]:428: in function <[string "boot.lua"]:274>
    [C]: in function 'xpcall'
```

I haven't found the cause of the first problem. The second one seems related to character case (note how the error message references `GLOB/LOGIN.LUA`, yet `glob` is in lower case in the save directory):

```
$ ls ~/.local/share/love/BallPlayCupid/
config  glob  homemadepuzzles  users
```

EDIT: and login.lua inside the glob directory is in lower case too:

```
$ ls ~/.local/share/love/BallPlayCupid/glob/
login.lua
```

**Comment by Tricky1975** [comment]
Thank you.
I saw the same bug in Windows, and considering the most likely nature of this bug this will happen on Mac as well. I'm on it to get this fixed.

EDIT: Source of the bug has been found. Fixing it now. After close this as fixed I will upload a new version with this bug fixed, but I need to note this can take a few hours.
- [x] In game crash
- [x] Linux login crash case sensitivity issue in glob/login.lua

**Comment by Tricky1975** [comment]
The case sensitiviy bug has been fixed now, but as I need loads of time to get Linux on the move (read disconnect my Mac Mini and connect my Linux machine to all the machines my Mac is now connected to) I cannot yet test this until I got more time. Anyway, if this happens again, only the glob folder will have to be removed (I hope). If not, please tell me in a new issue so I can sort this out.

I will now upload the new version. The fixed version will be 16.05.01, so any number below that still contains this bug.

**Comment by ghost** [comment]
Thanks for the quick fix! I'm getting a new crash now with the new version, right when selecting a level. I'll open a new report.
